src/main/resources/db/changelog/changes/04-create-user_role-table.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user_role-table
+      author: oleg
+      changes:
+        - createTable:
+            tableName: user_role
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

Comment:
add pk and fk constraint

Reply:
I think this is done automatically and I don't need do this

Reply:
Do you check it in your db? 
Better to ad them.